bugwarrior/services/taiga.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import absolute_import
+
+import requests
+import six
+
+from bugwarrior.db import CACHE_REGION as cache
+from bugwarrior.config import die
+from bugwarrior.services import IssueService, Issue
+
+
+class TaigaIssue(Issue):
+    SUMMARY = 'taigasummary'
+    URL = 'taigaurl'
+    FOREIGN_ID = 'taigaid'
+
+    UDAS = {
+        SUMMARY: {
+            'type': 'string',
+            'label': 'Taiga Summary'
+        },
+        URL: {
+            'type': 'string',
+            'label': 'Taiga URL',
+        },
+        FOREIGN_ID: {
+            'type': 'string',
+            'label': 'Taiga Issue ID'
+        },
+    }
+    UNIQUE_KEY = (URL, )
+
+    def to_taskwarrior(self):
+        return {
+            'project': self.extra['project'],
+            'annotations': self.extra['annotations'],
+            self.URL: self.extra['url'],
+
+            'priority': self.origin['default_priority'],
+            'tags': self.record['tags'],
+            self.FOREIGN_ID: self.record['ref'],
+            self.SUMMARY: self.record['subject'],
+        }
+
+    def get_default_description(self):
+        return self.build_default_description(
+            title=self.record['subject'],
+            url=self.get_processed_url(self.extra['url']),
+            number=self.record['ref'],
+            cls='issue',
+        )
+
+
+class TaigaService(IssueService):
+    ISSUE_CLASS = TaigaIssue
+    CONFIG_PREFIX = 'taiga'
+
+    def __init__(self, *args, **kw):
+        super(TaigaService, self).__init__(*args, **kw)
+        self.url = self.config_get('base_uri')
+        self.auth_token = self.config_get_password('auth_token', 'nousername')
+        self.label_template = self.config_get_default(
+            'label_template', default='{{label}}', to_type=six.text_type
+        )
+        self.session = requests.session()
```

Comment:
Yeah, try it:

```python
>>> import requests
>>> requests.Session(headers={})
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    requests.Session(headers={})
TypeError: __init__() got an unexpected keyword argument 'headers'
```

Also, check the [api changes section of the master docs](http://docs.python-requests.org/en/master/api/#api-changes) for an explicit mention.